pkg/health/condition/check_backup_ready.go

```diff
@@ -48,6 +50,12 @@ func (a *backupReadyCheck) Check(ctx context.Context, etcd druidv1alpha1.Etcd) R
 		message: "Cannot determine etcd backup status",
 	}
 
+	// Special case of etcd not being configured to take snapshots
+	// Do not add the BackupReady condition if backup is not configured
+	if etcd.Spec.Backup.Store == nil || etcd.Spec.Backup.Store.Provider == nil {
```

Comment:
Please add this condition as well.
```suggestion
	if etcd.Spec.Backup.Store == nil || etcd.Spec.Backup.Store.Provider == nil  || len(*etcd.Spec.Backup.Store.Provider)==0{
```

Reply:
Added